src/aavruti/MissingLink.java

```diff
@@ -0,0 +1,55 @@
+package aavruti;
+
+import java.util.*;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class MissingLink {
+
+	static WebDriver driverSetup() {
+
+		String osName = System.getProperty("os.name").toLowerCase();
+		String path = osName.contains("windows") ? "./resources/windows/chromedriver.exe" 
+				: osName.contains("mac") ? "./resources/mac/chromedriver" : null;
+		System.setProperty("webdriver.chrome.driver", path);
+		WebDriver driver = new ChromeDriver();
+		driver.manage().window().maximize();
+		return driver;
+	}
+
+	void findMissingLink() {
+		WebDriver driver = driverSetup();
+		driver.get("file:///G:/TechnoGitProject/SeleniumTechnoJun20/resources/forms/MissingLink.html");
+
+		if(driver.findElement(By.xpath("//p")).getText().equals("Missing Link")) {
```

Comment:
Here there is a no relation between p and a tag, so you should not right //a link logic under p tag.

Reply:
Removed p tag validation as per your given suggestion.